docs/examples/elements_plate.rst

```diff
@@ -0,0 +1,17 @@
+********************************************************************************
+Elements: Plate
+********************************************************************************
+
+A plate is a data structure that represents a pair of polygons. 
+
+The frames are oriented outwards. The first polygon is the base.
+The meshing is performed by the Ear clipping algorithm without considering the holes.
```

Comment:
Removed from description. I see that there are changes from compas_triangle to compas_gmsh. It would be great to have a short example for compas2 **contrained_delaunay_triangulation**.

At the current moment, I will just store properties for outlines, and the meshing can be done later.